run_kraken.py

```diff
@@ -189,15 +189,7 @@ def main(cfg):
                     scenario_type = list(scenario.keys())[0]
                     scenarios_list = scenario[scenario_type]
                     if scenarios_list:
-                        # Inject pod chaos scenarios specified in the config
-                        if scenario_type == "pod_scenarios":
```

Comment:
I think it would be good to still keep this as an error message for old config files.

Reply:
Restored the previous logic. I think that we should get rid of old config file support sooner or later 😅